web/controllers/auth.ex

```diff
@@ -0,0 +1,48 @@
+defmodule Notox.Auth do
+  import Plug.Conn
+
+  def init(_opts) do
+  end
+
+  defp get_current_user(conn) do
+    current_user_id = get_session(conn, :user_id)
+    current_user_id && Notox.Repo.get(Notox.User, current_user_id)
+  end
+
+  def call(conn, _opts) do
```

Comment:
`call` is one of the two functions every plug needs to implement. I'd consider putting it right under `init`, since it's what most people will be looking for when they open the plug